src/python/WMCore/WMSpec/StdSpecs/StdBase.py

```diff
@@ -875,8 +840,14 @@ def factoryWorkloadConstruction(self, workloadName, arguments):
 
         Named this way so that nobody else will try to use this name.
         """
-        self.masterValidation(schema=arguments)
-        self.validateSchema(schema=arguments)
+        if 'OriginalRequestName' not in arguments:
```

Comment:
For clone, we don't have validation function executed. (We will skip the validation so it should be fine). I think eventually I would like to add validation for only newly overwritten values not the ones which was cloned since that should be validated before (in original request creation). Are you actually seeing the validation fails. I didn't have problem when I clone the request.